kodilla-good-patterns/src/main/java/com/kodilla/kodillagoodpatterns/challenges/OrderRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.kodilla.kodillagoodpatterns.challenges;
+
+public class OrderRepository implements OrdersRepository{
+
+
+
+    @Override
```

Comment:
analogicznie jw. Jeden enter, nie 3. W razie czego Intelij robi to automatycznie Code > Reformat/Rearrange Code